theory/wp/countpairs_wp_impl.c.src

```diff
@@ -307,7 +307,9 @@ int countpairs_wp_DOUBLE(const int64_t ND, DOUBLE * restrict X, DOUBLE * restric
                                                                          boxsize, boxsize, boxsize,
                                                                          -1.0, rpmax, pimax,/*max_3D_sep, max_2D_sep, max_1D_sep*/
                                                                          options->enable_min_sep_opt,
-                                                                         options->autocorr, options->periodic);//options->autocorr == 1 and options->periodic == 1
+                                                                         options->autocorr,
+                                                                         options->periodic, options->periodic, options->periodic);
```

Comment:
Hmm `wp` has to be periodic along all axes. Not sure how to show that at this code level - some macro define/enum?

Reply:
We'll want to add a check to `wp` to make sure the box size and periodicity is the same in all dimensions.